deploy/run.sh

```diff
@@ -0,0 +1,8 @@
+/bin/sh
```

Comment:
I think this will just execute shell and never run rest of the script, if you intended to put a [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)#:~:text=In%20computing%2C%20a%20shebang%20is,bang%2C%20or%20hash%2Dpling.) then I think it needs to be prefixed by `#!`
```bash
#!/bin/sh
```